x-pack/plugins/security_solution/public/common/components/url_state/helpers.ts

```diff
@@ -23,15 +23,61 @@ import { timelineSelectors } from '../../../timelines/store/timeline';
 import { formatDate } from '../super_date_picker';
 import { NavTab } from '../navigation/types';
 import { CONSTANTS, UrlStateType } from './constants';
-import { ReplaceStateInLocation, KeyUrlState, ValueUrlState } from './types';
+import {
+  ReplaceStateInLocation,
+  KeyUrlState,
+  ValueUrlState,
+  UrlState,
+  UrlStateProps,
+  UrlStateStateToPropsType,
+} from './types';
 import { sourcererSelectors } from '../../store/sourcerer';
 import { SourcererScopeName, SourcererUrlState } from '../../store/sourcerer/model';
+import {
+  CASES_PATH,
+  DETECTIONS_PATH,
+  ALERTS_PATH,
+  RULES_PATH,
+  HOSTS_PATH,
+  NETWORK_PATH,
+} from '../../../../common/constants';
 
 export const isDetectionsPages = (pageName: string) =>
   pageName === SecurityPageName.alerts ||
   pageName === SecurityPageName.rules ||
   pageName === SecurityPageName.exceptions;
 
+// This list provides all the pages that display the security solution alert/events table which
+// launches the flyout when clicking expanded details
+
+const HOST_PAGE_EVENTS_PATH = `${HOSTS_PATH}/events`;
+const HOST_PAGE_EXTERNAL_ALERTS_PATH = `${HOSTS_PATH}/externalAlerts`;
+const DETECTION_RULES_DETAILS_PATH = `${RULES_PATH}/id`;
+const NETWORK_PAGE_EXTERNAL_ALERTS_PATH = `${NETWORK_PATH}/external-alerts`;
```

Comment:
currently we have management sections of the app using snake case, network pages using kebab casing, and host pages using snake casing. Should probably standardize that 🤣

Reply:
Umm.... 😂